pydoctor/templatewriter/util.py

```diff
@@ -1,19 +1,22 @@
 """Miscellaneous utilities."""
 
+import warnings
 from typing import Optional
-import os
+from pydoctor.model import Documentable
 
-from pydoctor import model
-from twisted.python.filepath import FilePath
-
-
-def srclink(o: model.Documentable) -> Optional[str]:
+def srclink(o: Documentable) -> Optional[str]:
     return o.sourceHref
 
-def templatefile(filename):
-    abspath = os.path.abspath(__file__)
-    pydoctordir = os.path.dirname(os.path.dirname(abspath))
-    return os.path.join(pydoctordir, 'templates', filename)
+def templatefile(filename: str) -> None:
+    """Deprecated: can be removed once Twisted stops using this."""
+    warnings.warn("pydoctor.templatewriter.util.templatefile() "
+        "is deprecated and returns None. It will be remove in future versions. "
+        "Please use the templating system.")
```

Comment:
I don't see this warning being printed when I generate Twisted's docs. Is it possible it is disabled by default, like most Python warnings?

Reply:
I think this functions do not get called at all anymore.